app/js/components/manageApps/AddonList.jsx

```diff
@@ -9,40 +9,48 @@
  */
 import React from 'react';
 
-export const AddonList = ({handleDownload, install, appList, openPage, openModal}) => {
+export const AddonList = ({handleDownload, appList, openPage, openModal}) => {
   return (
     <tbody>
       {
         appList.map((app, key) => {
           return (
             <tr key={key}>
-              <td onClick={() => openPage(app)}>
+              <td onClick={() => openPage(app.appDetails)}>
                 <img
-                  src={app.icons !== null && app.icons !== undefined ? 
-                    `/${location.href.split('/')[3]}/owa/openmrs-addonmanager/${app.icons[48]}` : 
+                  src={app.appDetails &&
+                    app.appDetails.icon !== null &&
+                    app.appDetails.icons !== undefined ? 
+                    `/${location.href.split('/')[3]}/owa/openmrs-addonmanager/${app.appDetails.icons[48]}` : 
                     `./img/omrs-button.png`}
                   alt="addon logo"
                 />
               </td>
-              <td onClick={() => openPage(app)}>
-                <div className="addon-name">{app.name}</div>
-                <div><h5  className="addon-description">{app.description}</h5></div>
+              <td onClick={() => openPage(app.appDetails)}>
+                <div className="addon-name">{app.appDetails ? app.appDetails.name : ''}</div>
```

Comment:
Curious question: I see here that you return either a name or an empty string dependent on the evaluation of app.appDetails. Why did you opt to return an empty string?

Reply:
Hey @LWanjiru It was a quick fix to resolve a bug, working on the update. Thanks for spotting it out.